src/Microsoft.AspNetCore.SignalR.Core/HubClientsWrapperThing.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public class HubClientsWrapperThing : IHubCallerClients
```

Comment:
Naming suggestions?

Reply:
`HubClientsWrapperThingy`

Reply:
Just `HubCallerClients` would be fine actually.

Reply:
Maybe this should be pubternal? I could maybe see reasons for users to want to create custom ones (when using HubContext directly for example) but I don't think we want to commit this to public API.